debian/compat

```diff
@@ -0,0 +1 @@
+7
```

Comment:
debian maintainer guide says you should set it to 10.

btw, I didn't know the "+NUMBER" syntax was valid, do you have documentation about it?

Reply:
Oh yes, I am not a debian dev, and I don't take the time to update myself with their ("news") packaging method.